pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2178,9 +2178,15 @@ ParticipantStatusTrackingInfo:
     - kits
   properties:
     medicalRecord:
-      $ref: '../pepper.yml#/components/schemas/ParticipantStatusTrackingInfo.Record'
+      type: array
+      minItems: 0
+      items:
+        $ref: '../pepper.yml#/components/schemas/ParticipantStatusTrackingInfo.Record'
     tissueRecord:
-      $ref: '../pepper.yml#/components/schemas/ParticipantStatusTrackingInfo.Record'
+      type: array
+      minItems: 0
+      items:
+        $ref: '../pepper.yml#/components/schemas/ParticipantStatusTrackingInfo.Record'
```

Comment:
If this is just adding workflows to the status, why are these changes here? This breaks multiple clients.

Reply:
It's not just adding workflows. As we discussed in the slack channel I mentioned in the description, we are going to switch to ES entirely for participant status. Previously there was strange aggregation of multiple records (see SQL query I put the reference to), now we're getting all records from ES. Now we have two ways:
1) pass all the records to clients (that breaks clients)
2) do the same strange aggregation on DSS side
Let's discuss

Reply:
How is the data expected to change? If a client was getting a single record now, what does the response look like now?

Reply:
This is not what I expected. Yes, we agreed to have DSM write out data into ES and that DSS should read data from ES instead of calling DSM's API. However, I think it is best that we keep the API the same. This means internally DSS should "massage" the data into the same format that we have now. In the Slack conversation, you'll see that I've noted this and have asked Simone about the expected behavior there. Overall, I think the only that should be changing in the API is a new `workflows` list.

Reply:
@yufengwng Can you check aggregation in DSM SQL queries? It seems meaningless. Could you confirm I should aggregate data the same way?
For example: org.broadinstitute.dsm.db.ParticipantStatus#SQL_SELECT_MR_STATUS

Reply:
@YuriyPavlovBI Based on my brief reading of that SQL, it looks like the query is aggregating medical records by participant id, and taking the minimum of the `requested`/`received` dates. This is consistent with what Simone described in the Slack conversation:
* Basically, a participant has a list of medical/tissue records.
* Each record is for a different institution.
* The way DSM builds the response of a single record is by taking the minimum date, _regardless_ of which institution is it.
I think we can just do the same in DSS -- read the `medicalRecord` or `tissueRecord` list and reduce it to a single object by taking the minimum dates.

Reply:
This is the example that Simone gave in the Slack conversation:
```
institution a: requested 01/01/2020
                received: ---
institution b: requested 02/01/2020
                received: 05/10/2020
would result in requested 01/01/2020 and received 05/10/2020
```

Reply:
It's sounding like none of the API beyond the `workflows` is expected to change- is this an accurate understanding, @yufengwng?

Reply:
Yes, my understanding is that the API should stay the same except for the `workflows` piece. In fact, I think we already have the `workflows` in there already. The original issue we were trying to solve was to workaround the authentication issue between DSS and DSM. So idea was to have DSM put data into ES so we don't have to do the auth dance. But folks got a little too enthusiastic about ES so we have to do some more refactors on DSS side when reading data out of ES.

Reply:
Good to know- thank you!

Reply:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/71830706/116559153-cda66a00-a908-11eb-97e9-0797b44c95b1.png">